Unify the style of the TWC articles (#2178)
ppy/osu-wiki: wiki/Tournaments/TWC/2011/en.md

---
tags:

> - TWC2011

Reviewer: ```suggestion
- 'TWC #1'
- TWC#1
- TWC 2011
- TWC2011
```

---
# Taiko World Cup 2011

![TWC 2011 Logo](img/logo.png)

The **Taiko World Cup 2011** (***TWC 2011***) was a country-based osu!taiko tournament hosted by the [osu! team](/wiki/People/The_Team). It was the first installment of the osu!taiko World Cup.

## Tournament Schedule

| Event | Timestamp |
| --: | :-- |
| Registration Phase | 2011-03-12/2011-04-09 |
| Group Stage - Week 1 | 2011-04-15/2011-04-17 |
| Group Stage - Week 2 | 2011-04-22/2011-04-26 |
| Group Stage - Week 3 | 2011-04-29/2011-05-01 |
| Quarterfinals | 2011-05-07/2011-05-08 |
| Semifinals | 2011-05-14/2011-05-15 |
| Finals | 2011-05-22/2011-05-29 |

## Prizes

| Placing | Prize(s) |
| :-: | :-- |
| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 3 months of osu!supporter, unique profile badge |
| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 2 months of osu!supporter |
| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 1 month of osu!supporter |

> | *Special* | 1 month of osu!supporter |

Reviewer 2: ```suggestion
| *Special* | 1 month of osu!supporter tag |
```

- TWC 2011
- TWC2011
---
# Taiko World Cup 2011

![TWC 2011 Logo](img/logo.png)

> The **osu!taiko World Cup 2011** (***TWC 2011***) was a country-based osu!taiko tournament hosted by the [osu! team](/wiki/People/The_Team). It was part of the 1st installment of the osu!taiko World Cup.

Reviewer 2: Not sure about the `part of`, since it seems complete (also logo has `The First Half`, so idk).
```suggestion
The **Taiko World Cup 2011** (***TWC 2011***) was a country-based osu!taiko tournament hosted by the [osu! team](/wiki/People/The_Team). It was the first installment of the osu!taiko World Cup.
```

Author: I'll take your suggestion instead.

## Tournament Schedule

| Event | Timestamp |
| --: | :-- |
| Registration Phase | 2011-03-12/2011-04-09 |
| Group Stage - Week 1 | 2011-04-15/2011-04-17 |
| Group Stage - Week 2 | 2011-04-22/2011-04-26 |
| Group Stage - Week 3 | 2011-04-29/2011-05-01 |
| Quarterfinals | 2011-05-07/2011-05-08 |
| Semifinals | 2011-05-14/2011-05-15 |
| Finals | 2011-05-22/2011-05-29 |

## Prizes

| Placing | Prize(s) |
| :-: | :-- |

> | ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 3 months of osu!supporter, unique profile badge |

Reviewer 2: There are instances with `tag` and others without it.

| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 2 months of osu!supporter |
| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 1 month of osu!supporter |
| *Special* | 1 month of osu!supporter |

![TWC 2011 Winner Badge](img/badge.png)

## Organization

> The osu!taiko World Cup 2011 was run by various community members by distributing the multitude of tasks into various fields of responsibility.

Reviewer 2: ```suggestion
The Taiko World Cup 2011 was run by various community members by distributing the multitude of tasks into various fields of responsibility.
```

## Participants

|  | Team | Members |
| :-: | :-: | :-- |
| ![][flag_BR] ![][flag_CL] | **Brazil-Chile** | **[Hyguys](https://osu.ppy.sh/users/362851)**, [Kurisuchianu](https://osu.ppy.sh/users/194345), [Foster Don](https://osu.ppy.sh/users/144464), [JhowM](https://osu.ppy.sh/users/599290) |
| ![][flag_CN] | **China** | **[Yowane \_ Haku](https://osu.ppy.sh/users/255267)**, [yukey](https://osu.ppy.sh/users/81972), [sagelord](https://osu.ppy.sh/users/56200), [small-qiang](https://osu.ppy.sh/users/619179) |
| ![][flag_FI] | **Finland** | **[XZero94](https://osu.ppy.sh/users/391366)**, [LunaticMara](https://osu.ppy.sh/users/194294), [Orkel](https://osu.ppy.sh/users/39385), [Pallero](https://osu.ppy.sh/users/7381) |

> | ![][flag_FR] | **France** | [\_Gezo\_](https://osu.ppy.sh/users/481582), [omegaflo](https://osu.ppy.sh/users/83291), Dagonpater, Didou |

Reviewer: ```suggestion
| ![][flag_FR] | **France** | [\_Gezo\_](https://osu.ppy.sh/users/481582), [omegaflo](https://osu.ppy.sh/users/83291), [Dagonpater](https://osu.ppy.sh/users/205257), [Didou](https://osu.ppy.sh/users/594822) |
```

| ![][flag_DE] | **Germany A** | **[OnosakiHito](https://osu.ppy.sh/users/290128)**, [DarkDunskin](https://osu.ppy.sh/users/98985), [RockeDon](https://osu.ppy.sh/users/624957), [Trashy](https://osu.ppy.sh/users/55191) |

> | ![][flag_DE] | **Germany B** | **[BrokenArrow1](https://osu.ppy.sh/users/384034)**, [Zeitgeist](https://osu.ppy.sh/users/227765), Der\_Locher, [bennius](https://osu.ppy.sh/users/3129206) |

Reviewer: ```suggestion
| ![][flag_DE] | **Germany B** | **[BrokenArrow1](https://osu.ppy.sh/users/384034)**, [Zeitgeist](https://osu.ppy.sh/users/227765), [Der_Locher](https://osu.ppy.sh/users/587835), [bennius](https://osu.ppy.sh/users/3129206) |
```

| ![][flag_HK] | **Hong Kong** | **[aabc271](https://osu.ppy.sh/users/155707)**, [XeBi320](https://osu.ppy.sh/users/342622), [KanaRin](https://osu.ppy.sh/users/310747), [Kanpakyin](https://osu.ppy.sh/users/394326) |
| ![][flag_IT] ![][flag_NL] | **Italy-Netherlands** | **[NeoRainier](https://osu.ppy.sh/users/326049)**, [Pheon](https://osu.ppy.sh/users/292295), [Lokamp](https://osu.ppy.sh/users/63671), [Henkie](https://osu.ppy.sh/users/16944) |
| ![][flag_JP] | **Japan** | **[inaba9510](https://osu.ppy.sh/users/477744)**, [TTTL](https://osu.ppy.sh/users/164583), [aruru1234](https://osu.ppy.sh/users/228812), [03123456789](https://osu.ppy.sh/users/295488) |

> | ![][flag_KR] | **Korea A** | **[KRZY](https://osu.ppy.sh/users/114017)**, BloodyLove, [Remilia-Scarlet](https://osu.ppy.sh/users/602783), [\[Shalon\]](https://osu.ppy.sh/users/317802) |

Reviewer: ```suggestion
| ![][flag_KR] | **Korea A** | **[KRZY](https://osu.ppy.sh/users/114017)**, [BloodyLove](https://osu.ppy.sh/users/139827), [Remilia-Scarlet](https://osu.ppy.sh/users/602783), [\[Shalon\]](https://osu.ppy.sh/users/317802) |
```

| ![][flag_KR] | **Korea B** | **[samiljul](https://osu.ppy.sh/users/156215)**, [Nyan](https://osu.ppy.sh/users/36102), [kds7769](https://osu.ppy.sh/users/257977), [Kim Jin Seok](https://osu.ppy.sh/users/58034) |

> | ![][flag_MY] | **Malaysia** | **[HeatKai](https://osu.ppy.sh/users/332555)**, mekadon, [ffyong](https://osu.ppy.sh/users/168154), [taiko_maniac1811](https://osu.ppy.sh/users/595764) |

Reviewer: ```suggestion
| ![][flag_MY] | **Malaysia** | **[HeatKai](https://osu.ppy.sh/users/332555)**, [mekadon](https://osu.ppy.sh/users/157835), [ffyong](https://osu.ppy.sh/users/168154), [taiko_maniac1811](https://osu.ppy.sh/users/595764) |
```

## Participants

|  | Team | Member |
| :-: | :-: | :-- |
| ![][flag_BR] ![][flag_CL] | **Brazil-Chile** | **[Hyguys](https://osu.ppy.sh/users/362851)**, [Kurisuchianu](https://osu.ppy.sh/users/194345), [Foster Don](https://osu.ppy.sh/users/144464), [JhowM](https://osu.ppy.sh/users/599290) |
| ![][flag_CN] | **China** | **[Yowane \_ Haku](https://osu.ppy.sh/users/255267)**, [yukey](https://osu.ppy.sh/users/81972), [sagelord](https://osu.ppy.sh/users/56200), [small-qiang](https://osu.ppy.sh/users/619179) |
| ![][flag_FI] | **Finland** | **XZero94**, [LunaticMara](https://osu.ppy.sh/users/194294), [Orkel](https://osu.ppy.sh/users/39385), [Pallero](https://osu.ppy.sh/users/7381) |

> | ![][flag_FR] | **France** | [\_Gezo\_](https://osu.ppy.sh/users/481582), [omegaflo](https://osu.ppy.sh/users/83291), [Dagonpater](https://osu.ppy.sh/users/Dagonpater), [Didou](https://osu.ppy.sh/users/Didou) |

Author: basically just plain text right?

> | ![][flag_FR] | **France** | [\_Gezo\_](https://osu.ppy.sh/users/481582), [omegaflo](https://osu.ppy.sh/users/83291), [Dagonpater](https://osu.ppy.sh/users/Dagonpater), [Didou](https://osu.ppy.sh/users/Didou) |

Reviewer 3: Yeah, specially if some of those can point to different unrelated users.

| ![][flag_GB] | **United Kingdom** | **[Rokodo](https://osu.ppy.sh/users/23374)**, [Sander-Don](https://osu.ppy.sh/users/284307), [MMzz](https://osu.ppy.sh/users/128993), [Nazo](https://osu.ppy.sh/users/55106) |
| ![][flag_US] | **United States** | **[OzzyOzrock](https://osu.ppy.sh/users/465153)**, [KYOUI](https://osu.ppy.sh/users/460978), [Backfire](https://osu.ppy.sh/users/263110), [anongos](https://osu.ppy.sh/users/7135) |

## Groups

| Group A | Group B | Group C | Group D |
| :-: | :-: | :-: | :-: |
| ![][flag_CN] China | ![][flag_FR] France | ![][flag_BR] ![][flag_CL] Brazil-Chile | ![][flag_FI] Finland |
| ![][flag_DE] Germany A | ![][flag_KR] South Korea A | ![][flag_HK] Hong Kong | ![][flag_DE] Germany B |
| ![][flag_IT] ![][flag_NL] Italy-Netherlands | ![][flag_TW] Taiwan | ![][flag_PL] Poland | ![][flag_JP] Japan |
| ![][flag_MY] Malaysia | ![][flag_GB] United Kingdom | ![][flag_US] United States | ![][flag_KR] South Korea B |
| ![][flag_PH] Philippines |  |  |  |

------------------------------------------------------------------------

![TWC 2011 Podium](img/podium.png)

------------------------------------------------------------------------

> ## Mappools

Reviewer 2: not applied
